### PR TITLE
Simple bash script to help manage the secrets.py file

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: backend/setup_secrets.bash [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
```
Comment (DolceTriade) [inline backend/setup_secrets.bash:2]: set -e

File: backend/setup_secrets.bash [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+if ! [[ -r "$conf" ]]; then
+  printf 'Could not read the conf file, %s.' "$conf"
+  exit 1
+fi
+
+dest="${0%/*}"
+[[ "$0" != */* ]] && dest='./'
+cp "$conf" "$dest/secrets.py"
+
+printf 'Done! Creeated %s and copied it to %s\n' "$conf" "$dest/secrets.py"
```
Comment (DolceTriade) [inline backend/setup_secrets.bash]: Created

File: backend/setup_secrets.bash [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  } > "$conf"
+
+fi
+
+if ! [[ -r "$conf" ]]; then
+  printf 'Could not read the conf file, %s.' "$conf"
+  exit 1
+fi
+
+dest="${0%/*}"
```
Comment (DolceTriade) [inline backend/setup_secrets.bash:48]: This is fancy, but perhaps just consider using $(dirname $0) in the interest of readability?

Comment (IsaacG) [reply backend/setup_secrets.bash:48]: This is POSIX paramter expansion and not uncommon. It's also faster and the recommended approach.

File: backend/setup_secrets.bash (hunk 1 above)
Comment (IsaacG) [reply backend/setup_secrets.bash:2]: See other PR with issues with set -e

File: backend/setup_secrets.bash (hunk 2 above)
Comment (IsaacG) [reply backend/setup_secrets.bash]: Fixed

**Review by DolceTriade — APPROVED** [review]

**Pull request merged**